src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -26,9 +26,6 @@ def get(self, request, project):
         """
         Fetches alert rules and legacy rules for an organization
         """
-        if not features.has("organizations:incidents", project.organization, actor=request.user):
```

Comment:
@iProgramStuff This should be good to remove the feature flag yeah? It's the combined rules listing.

Reply:
Should be good :-)